### ISSUE TITLE
Getting an IP address using the POST method with data returns null.

### ISSUE THREAD

**Issue by mavyfaby** [elysiajs/elysia#283]
Consider this example:

```ts
import { Elysia } from "elysia";

const app = new Elysia();
const port = 3000;

app.post("/", context => {
  return `Your IP address: ${app.server?.requestIP(context.request) || null}`;
});

app.listen(port, () => {
  console.log(`Running on port ${port}`);
});
```

When I request with a POST method that includes a data `curl -X POST http://localhost:3000 -d "hello=world"`. It doesn't return the IP address.

```ts
Your IP address: null
```

Bun when I remove the data `curl -X POST http://localhost:3000`, it's returning the IP address as expected:

```sh
Your IP address: { address, family, port }
```

P.S. I tested it with the native 'Bun.serve' and it works perfectly, however it doesn't work on Elysia.

**Comment by SaltyAom** [comment]
Blocking on https://github.com/oven-sh/bun/issues/6613

**Comment by kravetsone** [comment]
[Fixed by Bun@1.1.27](https://bun.sh/blog/bun-v1.1.27#fixed-server-requestip-after-await)

```ts
import { Elysia } from "elysia";

const app = new Elysia();
const port = 3000;

app.post("/", (context) => {
    return `Your IP address: ${
        app.server?.requestIP(context.request)?.address || null
    }`;
});

app.listen(port, () => {
    console.log(`Running on port ${port}`);
});

```
![image](https://github.com/user-attachments/assets/5cdfde26-e9c4-441f-acf0-81a12fe0c57c)

**Comment by mavyfaby** [comment]
I confirmed this now works! I'll now close the issue.
